packages/react-aria-components/src/Select.tsx

```diff
@@ -151,7 +152,7 @@ function SelectValue<T extends object>(props: SelectValueProps<T>, ref: Forwarde
   let renderProps = useRenderProps({
     ...props,
     // TODO: localize this.
-    defaultChildren: rendered || 'Select an item',
+    defaultChildren: rendered || placeholder || 'Select an item',
```

Comment:
I thought `Select an item` was going to be a translation, but it looks like we aren't doing default translations.